deno/deepgram-language-detection/README.md

```diff
@@ -0,0 +1,65 @@
+# ⚡ deepgramLanguageDetection()
+
+A Deno Cloud Function for detecting language from audio using [Deepgram](https://deepgram.com/)
+
+_Example function payload:_
+
+```json
+{
+  "fileUrl": "https://static.deepgram.com/examples/interview_speech-analytics.wav"
+}
+```
+
+_Successful function response::_
+
+```json
+{ "deepgramData": {}, "success": true }
+```
+
+_Error function response:_
+
+```json
+{ "message": "error message", "success": false }
+```
+
+## 📝 Variables
+
+List of variables used by this cloud function:
+
+**DEEPGRAM_API_KEY** - Your Deepgram API key.
+
+## 🚀 Deployment
+
+1. Clone this repository, and enter this function folder:
+
+```
+$ git clone https://github.com/open-runtimes/examples.git && cd examples
+$ cd deno/deepgram-language-detection
+```
+
+2. Enter this function folder and build the code:
+
+```
+docker run -e INTERNAL_RUNTIME_ENTRYPOINT=src/mod.ts --rm --interactive --tty --volume $PWD:/usr/code openruntimes/deno:v2-1.21 sh /usr/local/src/build.sh
+```
+
+As a result, a `code.tar.gz` file will be generated.
+
+3. Start the Open Runtime:
+
+```
+docker run -p 3000:3000 -e INTERNAL_RUNTIME_ENTRYPOINT=src/mod.ts -e INTERNAL_RUNTIME_KEY=secret-key --rm --interactive --tty --volume $PWD/code.tar.gz:/tmp/code.tar.gz:ro openruntimes/deno:v2-1.21 sh /usr/local/src/start.sh
+```
+
+Your function is now listening on port `3000`, and you can execute it by sending `POST` request with appropriate authorization headers. To learn more about runtime, you can visit Deno runtime [README](https://github.com/open-runtimes/open-runtimes/tree/main/runtimes/deno-1.21).
+
+4. Curl Command
+
+```bash
+curl http://localhost:3000/ -d '{"variables": {"DEEPGRAM_API_KEY":"<YOUR_DEEPGRAM_API_KEY>"},"payload": "{\"fileUrl\":\"https://static.deepgram.com/examples/interview_speech-analytics.wav\"}"}' -H "X-Internal-Challenge: secret-key" -H "Content-Type: application/json"
```

Comment:
@gewenyu99 @Rinshin-Jalal I would actually prefer this to be passed as a string rather than JSON object. That said, the code doesn't support the data passed as a string.

Reply:
@Rinshin-Jalal please make sure to [request a re-review](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/requesting-a-pull-request-review) after testing your code.

Reply:
> @gewenyu99 @Rinshin-Jalal I would actually prefer this to be passed as a string rather than JSON object to make it compatible with Appwrite Functions. That said, the code doesn't support the data passed as a string.

I can't handle that because it is handled by openruntimes itself we don't need too..

Data passed as string is also working

Reply:
@gewenyu99, can this be resolved now?